app/register/page.tsx

```diff
@@ -111,6 +114,9 @@ const Register = (): JSX.Element => {
       router.push('/weeklyPicks');
     } catch (error) {
       console.error('Registration Failed', error);
+      toast.custom(
+        <Alert variant={AlertVariants.Error} message="Something went wrong." />,
+      );
     }
```

Comment:
is there a toast for successfully registering?

Reply:
I can add it in, but when you register successfully it just redirects you to /weeklyPicks. I can find a way to add it in so it pops up when you're on the /weeklyPicks page after registering